lib/stdlib/doc/src/binary.xml

```diff
@@ -256,6 +256,25 @@
       </desc>
     </func>
 
+    <func>
+      <name name="encode_hex" arity="2" since="OTP 26.0"/>
```

Comment:
You could conflate the docs for the two functions `encode_hex/1` and `encode_hex/2` into one, and mention in the description that `uppercase` is the default.